### PR TITLE
[MPLUGIN-497] Upgrade components

### PR DESCRIPTION
**Pull request by michael-o**

### CODE REVIEW

File: maven-script/maven-script-ant/pom.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@ under the License.
     <dependency>
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-ant-factory</artifactId>
+      <!-- Do not upgrade, ompilation will fail! -->
```
Comment (slawekjaranowski) [inline maven-script/maven-script-ant/pom.xml:64]: `compilation`?

**Review by slawekjaranowski — COMMENTED** [review]
I can't see how those dependencies are used in ITs

**Comment by michael-o** [comment]
> I can't see how those dependencies are used in ITs

True, I used my generic summary. Will fix.

**Review by slawekjaranowski — COMMENTED** [review]
please also upgrade PR title 😄

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**